notebooks/demo2/transformer-inference-sparse.ipynb

```diff
@@ -0,0 +1,1410 @@
+{
```

Comment:
<p>The original model bin (farm bert based model) has the size 475.6MB, inference time of 6.5min and f1 score of 0.9156</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/os-climate/aicoe-osc-demo/pull/234/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='46'/>

Reply:
Thank you Shrey!